src/scss/components/_carousel.scss

```diff
@@ -1,220 +1,242 @@
-      @extend %transform-positive;
-    }
-
-    @include carousel-item-level('.active', '.carousel-item', 5) {
-      display: block;
-    }
-
-    @include carousel-item-level('.carousel-item.active:not(.carousel-item-right):not(.carousel-item-left)', '.carousel-item:not(.carousel-item--last)', 5) {
-      transition: none;
-    }
-
-    @include carousel-item-level('.carousel-item-next.carousel-item-left', '.carousel-item', 5, false) {
-      @extend %transform-negative;
-    }
-
-    @include carousel-item-level('.carousel-item-prev.carousel-item-right', '.carousel-item', 5, false) {
-      @extend %transform-positive;
-    }
-
-    .carousel-item-prev.carousel-item-right {
-      position: absolute;
-      top: 0;
-      left: 0;
-      z-index: -1;
-      display: block;
-      visibility: visible;
-    }
-  }
-
-  &--centered {
-    overflow: hidden;
-
-    .carousel-inner {
-      overflow: visible;
-      transform: translateX(-62.5%);
-
-      @include media-breakpoint-up(md) {
-        transform: translateX(-12.5%);
-
-        .active + .carousel-item + .carousel-item + .carousel-item + .carousel-item + .carousel-item {
-          display: block;
-        }
-
-        .carousel-item {
-          &--last {
-            transform: translate3d(100%, 0, 0);
-          }
-        }
-      }
-    }
-  }
+	&--fixed-height {
+		overflow: hidden;
+		height: 11.875em;
+
+		@include media-breakpoint-up(lg) {
+			height: 25em;
+		}
+
+		.carousel-inner,
+		.carousel-item,
+		figure,
+		img {
+			height: 100%;
+		}
+
+		img {
+			object-fit: cover;
+		}
+	}
+
+	&--sidebar {
+		.carousel-item {
+			@include media-breakpoint-up(lg) {
+				display: block;
+				float: none;
+			}
+		}
+	}
+
+	&--loop {
+		.carousel-item {
+			margin-right: 0;
+		}
+
+		.carousel-item-next,
+		.carousel-item-prev {
+			position: relative;
+			transform: translate3d(0, 0, 0);
+		}
+
+		.carousel-item-prev:not(.carousel-item-right) {
+			transform: translateX(-100%);
+		}
+
+		.active.carousel-item-left + .carousel-item-next.carousel-item-left {
+			@extend %transform-negative;
+		}
+
+		.active.carousel-item-right + .carousel-item-prev.carousel-item-right {
+			@extend %transform-positive;
+		}
+
+		@include carousel-item-level('.active', '.carousel-item', 7) {
+			display: block;
+		}
+
+		@include carousel-item-level('.carousel-item.active:not(.carousel-item-right):not(.carousel-item-left)', '.carousel-item:not(.carousel-item--last)', 7) {
+			transition: none;
+		}
+
+		@include carousel-item-level('.carousel-item-next.carousel-item-left', '.carousel-item', 7, false) {
+			@extend %transform-negative;
+		}
+
+		@include carousel-item-level('.carousel-item-prev.carousel-item-right', '.carousel-item', 7, false) {
+			@extend %transform-positive;
+		}
+
+		.carousel-item-prev.carousel-item-right {
+			position: absolute;
+			top: 0;
+			left: 0;
+			z-index: -1;
+			display: block;
+			visibility: visible;
+		}
+
+		&[data-slide-number="5"] {
+			.col-md-3 {
```

Comment:
if you want to use this just remove .col-md-3 from as class of the element and apply the rule on carousel-item